crates/arroyo-controller/src/schedulers/mod.rs

```diff
@@ -329,6 +345,11 @@ impl NodeScheduler {
         }
     }
 
+    async fn client(node: &NodeStatus) -> anyhow::Result<NodeGrpcClient<Channel>> {
+        let channel = connect_grpc(format!("http://{}", node.addr), &config().node.tls).await?;
```

Comment:
Should this be `https://` for TLS enabled services?

Reply:
Maybe the `http://` part should just be part of the `connect_grpc` responsibility then?  (i.e. instead of the URL, you pass the address, path (defaulted to ""), and tls config?).

Reply:
That could be a good option, but would be a breaking change (or some version of this at each callsite) to transition to. The other callers of this function are passing in user-configure endpoints, which all include the scheme.